### ISSUE TITLE
Warn about insufficient contrast ratio with a gradient background

### ISSUE THREAD

**Issue by IchordeDionysos** [GoogleChrome/lighthouse#5998]
Tested with Lighthouse in Chrome Version 68.0.3440.106

**Feature request summary**

Lighthouse doesn't warn about insufficient contrast ratio with a gradient background.
Example page: 

```html
<!DOCTYPE html>
<html>
<head>
    <meta charset="utf-8" />
    <meta http-equiv="X-UA-Compatible" content="IE=edge">
    <title>Page Title</title>
    <meta name="viewport" content="width=device-width, initial-scale=1">
    <style>
        .card {
            height: 100px;
            width: 300px;
            border-radius: 8px;
            background: linear-gradient(44.3deg, #dddddd 0%, #eeeeee 100%);	
            color: rgba(255, 255, 255, 0.6);
        }
        .card p {
            height: 100%;
            padding: 40px 0;
            text-align: right;
        }
    </style>
</head>
<body>
    <div class="card">
        <p>Text</p>
    </div>
</body>
</html>
```

Lighthouse warnings with only Accessibility enabled: 
https://cl.ly/3c0ea732293b/Screenshot%202018-09-09%20at%2021.55.21.png

**What is the motivation or use case for changing this?**
When you use a lot of gradients on your page you may be not aware of the fact, that there are contrast ratio issues because lighthouse does not report those.

**How is this beneficial to Ligthhouse?**
I my opinion this critical for A11Y to work correctly

**Comment by justinribeiro** [comment]
Hi thanks for the report! Currently LH uses aXe-core behind the scenes to do accessibility auditing. Using aXe directly, I'm able to duplicate your issue using your provided sample. Upstream, there is an open ticket currently for gradient and alpha value background color contrast checking approach that I believe covers your aforementioned case: https://github.com/dequelabs/axe-core/issues/708

**Comment by brendankenny** [comment]
There are a lot of difficulties with contrast detection, and I'm inclined to think we shouldn't track them all in Lighthouse when there are explicit axe issue for them. Let's close and wait on https://github.com/dequelabs/axe-core/issues/708 for updates.
